broker-daemon/relayer/relayer-client.js

```diff
@@ -86,11 +87,20 @@ class RelayerClient {
 
     this.credentials = channelCredentials
 
-    this.makerService = caller(this.address, this.proto.MakerService, this.credentials, GRPC_STREAM_OPTIONS)
-    this.takerService = caller(this.address, this.proto.TakerService, this.credentials, GRPC_STREAM_OPTIONS)
-    this.orderBookService = caller(this.address, this.proto.OrderBookService, this.credentials, GRPC_STREAM_OPTIONS)
-    this.paymentChannelNetworkService = caller(this.address, this.proto.PaymentChannelNetworkService, this.credentials)
-    this.adminService = caller(this.address, this.proto.AdminService, this.credentials)
+    const makerServiceClient = new this.proto.MakerService(this.address, this.credentials, GRPC_STREAM_OPTIONS)
+    this.makerServiceClient = caller.wrap(makerServiceClient, {}, { interceptors: [grpcDeadlineInterceptor] })
```

Comment:
`this.makerServiceClient` can be renamed to `this.makerService`

Reply:
Makes me worried that this passed our test specs. Good catch @bradleystachurski